### PR TITLE
Initial "getting started" documentation.

### PR DESCRIPTION
**Pull request by lgruen**

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+At the moment, using Hail requires launching a Dataproc (Spark) cluster.
+Please always set a maximum age for the cluster (`--max-age` below), to avoid
+accidental spending in case you forget to stop the cluster after your job has
+completed:
+
+```bash
+hailctl dataproc start --max-age 2h --region australia-southeast1 my-cluster
+```
+
+If you're interested in the Hail internals, this developer focussed
```
Comment (pdiakumis) [inline getting_started.md:161]: That really is an awesome overview, was reading it earlier today. Really helps in mapping out the codebase.

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+[storage policies]: https://github.com/populationgenomics/storage-policies
+
+Please avoid storing any non-public data on your laptop, as that increases the
+risk of accidental data breaches significantly. Keep in mind that any
+non-public genomic data is highly sensitive.
+
+# Hail
+
+[Hail] is an amazing open source library and platform for genomic data
+analysis, developed at the Broad Institute. Given it's proven scalability and
```
Comment (pdiakumis) [inline getting_started.md:121]:
```suggestion
analysis, developed at the Broad Institute. Given its proven scalability and
```

Comment (lgruen) [reply getting_started.md:121]: Thanks!

File: getting_started.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+# Hail
+
+[Hail] is an amazing open source library and platform for genomic data
+analysis, developed at the Broad Institute. Given its proven scalability and
+our good relationship with the Hail development team, it's the Centre's main
+analysis platform.
+
+[Hail]: https://hail.is
+
+To install Hail, use the package in the CPG's Bioconda channel:
```
Comment (pdiakumis) [inline getting_started.md:127]:
```suggestion
To install Hail, use the package in the CPG's [conda channel]:
```

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+1.  Install [Miniconda].
+1.  Run the following to create a Conda environment called `hail`:
+    ```bash
+    conda create --name hail -c cpg -c bioconda -c conda-forge hail
+    conda activate hail
+    ```
+
```
Comment (pdiakumis) [inline getting_started.md:134]:
```suggestion
[conda channel]: https://anaconda.org/cpg
```

File: getting_started.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+[Hail] is an amazing open source library and platform for genomic data
+analysis, developed at the Broad Institute. Given its proven scalability and
+our good relationship with the Hail development team, it's the Centre's main
+analysis platform.
+
+[Hail]: https://hail.is
+
+To install Hail, use the package in the CPG's Bioconda channel:
+1.  Install [Miniconda].
+1.  Run the following to create a Conda environment called `hail`:
```
Comment (pdiakumis) [inline getting_started.md:129]:
```suggestion
1.  Run the following to create a conda environment called `hail`:
```

**Review by pdiakumis — APPROVED** [review]
Looks great Leo, thanks so much for setting this up!

**Comment by vladsaveliev** [comment]
That's an awesome "getting started"! I don't have any edits to suggest - could only add more stuff but in other PRs.

**Review by vladsaveliev — APPROVED** [review]

**Comment by lgruen** [comment]
Thanks very much for the reviews!

**Pull request merged**